### PR TITLE
[BEAM-14418] add navigation buttons to slider

### PR DESCRIPTION
**Pull request by bullet03**
[BEAM-14418] 
- add nav buttons to slider 
- change html sctucture of slider 

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by bullet03** [comment]
R: @nausharipov

File: website/www/site/assets/scss/_quotes.scss [hunk 1]
```diff
@@ -173,3 +172,16 @@
     }
   }
 }
+
+.swiper-container {
+  display: flex;
+  align-items: center;
+}
+
+.swiper-button-next-custom, .swiper-button-prev-custom {
```
Comment (nausharipov) [inline website/www/site/assets/scss/_quotes.scss:181]: format the file

File: website/www/site/layouts/index.html [hunk 2]
```diff
@@ -209,16 +209,24 @@ <h2 class="calendar-title">
...
+        <div class="swiper-wrapper">
+          {{ $data := index $.Site.Data .Site.Language.Lang }}
+          {{ range $quote := $data.quotes }}
+          <div class="swiper-slide">
+            {{ partial "quotes/quote.html" (dict "icon" $quote.icon "text" $quote.text  "logoUrl" $quote.logoUrl "linkUrl" $quote.linkUrl "linkText" $quote.linkText) }}
+          </div>
+          {{ end }}
         </div>
-        {{ end }}
+        <div id="swiper-pagination-id" class="swiper-pagination"></div>
```
Comment (nausharipov) [inline website/www/site/layouts/index.html:225]: can this div be shortened?

`<div id="swiper-pagination-id" class="swiper-pagination" / >`

File: website/www/site/layouts/index.html [hunk 3]
```diff
@@ -209,16 +209,24 @@ <h2 class="calendar-title">
...
+    <div class="swiper-container">
+      <div class="swiper-button-prev-custom">
+        <img src="/images/swiper-button-prev.png" alt="previous button">
+      </div>
+      <div class="quotes-desktop swiper">
+        <div class="swiper-wrapper">
+          {{ $data := index $.Site.Data .Site.Language.Lang }}
+          {{ range $quote := $data.quotes }}
+          <div class="swiper-slide">
+            {{ partial "quotes/quote.html" (dict "icon" $quote.icon "text" $quote.text  "logoUrl" $quote.logoUrl "linkUrl" $quote.linkUrl "linkText" $quote.linkText) }}
```
Comment (nausharipov) [inline website/www/site/layouts/index.html:221]: consider formatting the file

File: website/www/site/layouts/index.html (hunk 2 above)
Comment (bullet03) [reply website/www/site/layouts/index.html:225]: the layout somehow breaks, when i try to use the shortened variant <div id="swiper-pagination-id" class="swiper-pagination" />

File: website/www/site/layouts/index.html (hunk 3 above)
Comment (bullet03) [reply website/www/site/layouts/index.html:221]: tried to put attributes on different lines, but since it's hugo code doesn't work correctly

**Comment by dariabezkorovaina** [comment]
@pcoet Aizhamal asked to add arrows to the sides on desktop and she said they look good. Can you please give your LGTM? :)

**Review by pcoet — APPROVED** [review]
I didn't do any testing, but the changes LGTM.

**Comment by aromanenko-dev** [comment]
Is it ready to be merged?

**Comment by nausharipov** [comment]
> Is it ready to be merged?

yes

**Review by aromanenko-dev — CHANGES_REQUESTED** [review]
Just checked the commits of this PR, it requires some additional work to do before merge.

@bullet03 Could you squash all commits and rebase it on top of `master` branch? 

Also, please, avoid merging `master` branch into the feature branch and use `git rebase` instead. It will make commits history more clear.

**Comment by nausharipov** [comment]
> Just checked the commits of this PR, it requires some additional work to do before merge.
> 
> @bullet03 Could you squash all commits and rebase it on top of `master` branch?
> 
> Also, please, avoid merging `master` branch into the feature branch and use `git rebase` instead. It will make commits history more clear.

@aromanenko-dev we couldn't squash commits because there are merge commits among them. We'll cherry-pick those commits into a new branch, squash them and create a new PR.

**Comment by bullet03** [comment]
We have cherry-picked, squashed and rebased commits from this branch into the new branch: https://github.com/apache/beam/pull/17722

**Comment by aromanenko-dev** [comment]
@nausharipov 
> we couldn't squash commits because there are merge commits among them

Yes, this is why it's highly recommended to avoid merge commits and use `git rebase` instead.

**Pull request closed without merging**